backend/news/api/verify_token.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework_simplejwt.views import TokenVerifyView
+from rest_framework.response import Response
+from rest_framework import status
+
+
+class CustomTokenVerifyView(TokenVerifyView):
+    def post(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+
+        try:
+            serializer.is_valid(raise_exception=True)
+            return Response({"message": "Token is valid",
+                             "success": True}, status=status.HTTP_200_OK)
+        except Exception as e:
+            return Response({"message": "Token is invalid "
+                                        "or expired", "success": False},
+                            status=status.HTTP_401_UNAUTHORIZED)
```

Comment:
This could be done with simple TokenVerifyView, i did it to keep the consistency in responses of my application, default message response is : `{
    "code": "token_not_valid",
    "detail": "Token is invalid or expired"
}`

but my default is :

`{"message": "Token is valid",
                             "success": True}`